Description: Skip tests that fail on 32-bit architctures
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: https://github.com/ebkalderon/renderdoc-rs/issues/158
Last-Update: 2025-09-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/bindings.rs
+++ b/src/bindings.rs
@@ -219,6 +219,7 @@
     pub RemoveHooks: pRENDERDOC_RemoveHooks,
 }
 #[test]
+#[cfg(target_pointer_width = "64")]
 fn bindgen_test_layout_RENDERDOC_API_1_6_0__bindgen_ty_1() {
     const UNINIT: ::std::mem::MaybeUninit<RENDERDOC_API_1_6_0__bindgen_ty_1> =
         ::std::mem::MaybeUninit::uninit();
@@ -269,6 +270,7 @@
     pub SetCaptureFilePathTemplate: pRENDERDOC_SetCaptureFilePathTemplate,
 }
 #[test]
+#[cfg(target_pointer_width = "64")]
 fn bindgen_test_layout_RENDERDOC_API_1_6_0__bindgen_ty_2() {
     const UNINIT: ::std::mem::MaybeUninit<RENDERDOC_API_1_6_0__bindgen_ty_2> =
         ::std::mem::MaybeUninit::uninit();
@@ -319,6 +321,7 @@
     pub GetCaptureFilePathTemplate: pRENDERDOC_GetCaptureFilePathTemplate,
 }
 #[test]
+#[cfg(target_pointer_width = "64")]
 fn bindgen_test_layout_RENDERDOC_API_1_6_0__bindgen_ty_3() {
     const UNINIT: ::std::mem::MaybeUninit<RENDERDOC_API_1_6_0__bindgen_ty_3> =
         ::std::mem::MaybeUninit::uninit();
@@ -369,6 +372,7 @@
     pub IsTargetControlConnected: pRENDERDOC_IsTargetControlConnected,
 }
 #[test]
+#[cfg(target_pointer_width = "64")]
 fn bindgen_test_layout_RENDERDOC_API_1_6_0__bindgen_ty_4() {
     const UNINIT: ::std::mem::MaybeUninit<RENDERDOC_API_1_6_0__bindgen_ty_4> =
         ::std::mem::MaybeUninit::uninit();
@@ -413,6 +417,7 @@
     }
 }
 #[test]
+#[cfg(target_pointer_width = "64")]
 fn bindgen_test_layout_RENDERDOC_API_1_6_0() {
     const UNINIT: ::std::mem::MaybeUninit<RENDERDOC_API_1_6_0> = ::std::mem::MaybeUninit::uninit();
     let ptr = UNINIT.as_ptr();
